src/Language/Python/Syntax.hs

```diff
@@ -96,7 +96,6 @@ resolvePythonModules q = do
 
     search rootDir x = do
       trace ("searching for " <> show x <> " in " <> show rootDir)
-    -- JZ:TODO: Refactor later
       let path = normalise (rootDir </> normalise x)
```

Comment:
This is related to https://github.com/github/semantic/issues/537.  I will refactor it in another PR.